lib/promoCard/PromoCard.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+
+class PromoCard extends StatefulWidget {
+  final Function() playHome;
+  const PromoCard({Key? key, required this.playHome}) : super(key: key);
+
+  @override
+  State<PromoCard> createState() => _PromoCardState();
+}
+
+class _PromoCardState extends State<PromoCard> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: const EdgeInsets.only(top: 100.0),
+      alignment: Alignment.center,
+      decoration: const BoxDecoration(color: Color(0xFFFFEDF5)),
+      height: MediaQuery.of(context).size.height * 0.40,
```

Comment:
height is 35%

Reply:
changed the height of the promocard to 35%